vcpkg/users/platforms/android.md

```diff
@@ -40,24 +33,41 @@ There are four different Android ABIs, each of which maps to a vcpkg triplet. Th
 |---------------------------|----------------------|
 |arm64-android              | arm64-v8a            |
 |arm-android                | armeabi-v7a          |
+|arm-neon-android (w/neon)  | armeabi-v7a          |
 |x64-android                | x86_64               |
 |x86-android                | x86                  |
+|armv6-android              | armeabi              |
 
 ## Install libraries for Android using vcpkg
```

Comment:
Can we change this example to use manifest mode?

Reply:
done